src/assets/_project/_blocks/components/forms/qg-recaptcha.js

```diff
@@ -66,66 +64,49 @@ import keys from '../../data/qg-google-keys';
+     * Call submission handler api
+     * Get and display success message
      * @return {undefined}
      **/
-    footerFeedbackRecaptcha: function() {
-      var selfObj = this;
-      /**
-       * call recaptcha api to load script with the key on .qg-feedback-toggle click
-       **/
-      $('.qg-feedback-toggle').one('click', function(){
-        swe.ajaxCall(
-          'https://www.google.com/recaptcha/api.js?render=' + selfObj.footerFeedbackGoogleRecaptchaApiKey(),
-          'script',
-          onReady,
-          'Recaptcha unavailable',
-        );
-      });
-
-      /**
-       * onReady function is executed after calling recaptcha api in the above code
-       * This function creates a submit event
-       * Call submission handler api
-       * Get and display success message
-       * @return {undefined}
-       **/
-      let onReady = function (){
-        grecaptcha.ready(function () {
-          $('#qg-page-feedback-form').submit(function (event) {
-            var self = $(this);
-            event.preventDefault();
-            if ($('#qg-page-feedback-form li.invalid').length <= 0) {
-              var greptcha = selfObj.config.$feedbackForm.find('input[name="g-recaptcha-response"]');
-              var postUrl = self.attr('action');
-              var requestMethod = self.attr('method');
-              var $successMsgContainer = $('.thankyou');
-              grecaptcha.execute(selfObj.footerFeedbackGoogleRecaptchaApiKey(), {action: 'feedback'})
-                .then(function (token) {
-                  if (greptcha.length > 0) {
-                    greptcha.val(token);
-                    var formData = self.serialize();
-                    $.ajax({
-                      url: postUrl,
-                      type: requestMethod,
-                      data: formData,
-                      contentType: 'application/x-www-form-urlencoded',
-                      cache: false,
-                      processData: false,
-                    }).done(function (response) {
-                      $successMsgContainer.removeClass('d-none');
-                      $('#qg-page-feedback-form, .qg-feedback-toggle').addClass('d-none');
-                      $successMsgContainer.append($.parseHTML(JSON.parse(response).message));
-                    }).fail(function () {
-                      $successMsgContainer.append('<p>Request failed, please try again</p>');
-                    });
-                    return true;
-                  }
-                });
-            }
-          });
+    footerFeedbackSubmitWithRecaptchaCheck: function() {
+      var self = this;
+      grecaptcha.ready(() => {
+        $('#qg-page-feedback-form').submit(function (event) {
+          var targetFormSubmit = $(this);
+          event.preventDefault();
+          if ($('#qg-page-feedback-form li.invalid').length <= 0) {
+            var greptcha = self.config.$feedbackForm.find('input[name="g-recaptcha-response"]');
```

Comment:
This is a somewhat confusing variable name, especially with `grecaptcha` already existing.

Reply:
updated in the latest commit